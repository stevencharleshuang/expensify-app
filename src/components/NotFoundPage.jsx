import React from 'react';
import { Link } from 'react-router-dom';

export default function NotFoundPage() {
  return (
    <div>
      404!
      <br /><br />
      <Link to="/">Go Home</Link>
    </div>
  );
}

