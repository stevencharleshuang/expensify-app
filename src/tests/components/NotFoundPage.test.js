import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';
import toJSON from 'enzyme-to-json'
import ReactShallowRenderer from 'react-test-renderer/shallow';

test('should render ExpenseList with expenses', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
