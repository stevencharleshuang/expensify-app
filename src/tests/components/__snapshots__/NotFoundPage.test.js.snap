// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList with expenses 1`] = `
<div>
  404!
  <br />
  <br />
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
